src/main/kotlin/com/cognifide/gradle/aem/instance/InstancePlugin.kt

```diff
@@ -71,6 +71,11 @@ class InstancePlugin : AemPlugin() {
                 dependsOn(InstanceDown.NAME)
                 finalizedBy(InstanceUp.NAME)
             }
+            register<InstanceRestore>(InstanceRestore.NAME) {
```

Comment:
Restore as a separate task. Now you can be sure when you restore from a backup or you create an instance from scratch